### ISSUE TITLE
Implement USDC/USDCet rebalancing prototype

### ISSUE THREAD

**Issue by tobikuhlmann** [mento-protocol/abacus-prototype#27]
Implement a rebalancing prototype between Alfajores and Goerli to bridge and balance USDC and USDC of the reserve

Currently blocked by Abacus deploying on Goerli as The Portal Token Bridge is not on Kovan where Abacus is but on Goerli

**Comment by nvtaveras** [comment]
No longer relevant, closing
